tools/bug-report/pkg/content/content.go

```diff
@@ -148,6 +148,14 @@ func GetClusterContext() (string, error) {
 	return kubectlcmd.RunCmd("config current-context", "", false)
 }
 
+// GetNodeInfo returns node information.
+func GetNodeInfo(p *Params) (map[string]string, error) {
+	labels := "node-role.kubernetes.io/control-plane,node-role.kubernetes.io/master"
+	cmdStr := "describe nodes -l " + labels
```

Comment:
why do we need the filter?

Reply:
Ahh, it works without a filter too. as we need all nodes info no need to add a label filter. Thanks.